app/models/course.rb

```diff
@@ -39,6 +39,7 @@
 #  syllabus_file_size    :integer
 #  syllabus_updated_at   :datetime
 #  home_wiki_id          :integer
+#  timezone              :string           default("UTC")
```

Comment:
let's be sure to validate this against inclusion in the set ActiveSupport knows about.